packages/testing/src/jest-resolver.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+
+// ws workaround suggested in https://github.com/websockets/ws/issues/2171#issuecomment-1792147402
+// Mentioned solution has
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
```

Comment:
We can maybe clarify the comment, something like the following.

```suggestion
// Mentioned solution license is:
//    Copyright (c) Microsoft Corporation. All rights reserved.
//    Licensed under the MIT License.
```